library/src/scala/tasty/reflect/CompilerInterface.scala

```diff
@@ -1478,8 +1478,14 @@ trait CompilerInterface {
   def Definitions_Array_length: Symbol
   def Definitions_Array_update: Symbol
 
+  /** A dummy class symbol that is used to indicate repeated parameters
+   *  compiled by the Scala compiler.
```

Comment:
Can we clarify this or make an issue to clarify this in the future? Does this comment want to convey that this class is only used for Scalac repeated parameters?

Reply:
I copied the original comment to the interface. We can revise this documentation later.

Reply:
So can we make an issue for it? Or are you keeping track of such things in a different way?

Reply:
We will need to do a pass over everything in the API and probably rewrite/improve most of the documentation. Singling a few out does not seem to be worth it.